### ISSUE TITLE
SwitchDev to SAI adaptor

### ISSUE THREAD

**Issue by sequoiar** [dentproject/feature#43]
the mainstream switch ASIC vendors have already support OCP/SAI.
Is there any plan to add a SwitchDev to SAI adaptor layer ?

**Comment by sonoble** [comment]
Hello, your question is not clear. SwitchDev is part of the Linux kernel while SAI is a user space application. The only direction available is a SAI to SwitchDev conversion layer, but that would be of little value as you can just use SwitchDev directly.

**Comment by sequoiar** [comment]
@sonoble yes. OCP/SAI mean user-space. But, in meantime, ASIC vendor SDK can be built in linux kernel-space as well. So, if there is an existing SwitchDev to SAI API adapter layer, it's possible to avoid ASIC vender implement both SAI and SwitchDev API.
